pkgs/applications/misc/orca/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, pkgconfig, fetchurl, buildPythonApplication
+, autoreconfHook, wrapGAppsHook
+, intltool, yelp_tools, itstool, libxmlxx3
+, python, pygobject3, gtk3, gnome3
+, at_spi2_atk, at_spi2_core, pyatspi, dbus, dbus-python, pyxdg
+, louis ? null
```

Comment:
What is that supposed to be?

Reply:
louis or liblouis is a requirement to support braille, but it doesn't exist yet in nixpkgs (or I couldn't find it). One of the reasons it's WIP.

Reply:
A TODO comment next to `buildInputs` is probably a better place to mention it. Arguments for non-existing variables are quite confusing.